nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -1235,7 +1233,7 @@ def test_forward_rel_ops():
     test_forward_inception_v3()
     test_forward_inception_v1()
     test_forward_mobilenet()
-    test_forward_resnetv2()
+    #_test_forward_resnetv2()
```

Comment:
Why do you disable the resnetv2 test?

Reply:
Resnet V2 was enabled with GPU target earlier.
Tensorflow on CI doesn't support GPU now. Also I see some strange error with GPU in local setup.

Hence disabled now, I will consider it later.

Reply:
fixed this too.